<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клеточный автомат</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/mainresources.qrc">
    <normaloff>:/img/icons/main_icon</normaloff>:/img/icons/main_icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGraphicsView" name="gv_Field">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="12" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="pbRun">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Запустить</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mainresources.qrc">
          <normaloff>:/img/icons/play</normaloff>:/img/icons/play</iconset>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_scaleText">
        <property name="text">
         <string>Задать масштаб:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="SpeedBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="ScaleBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="pbStop">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Остановить</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mainresources.qrc">
          <normaloff>:/img/icons/stop</normaloff>:/img/icons/stop</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pb_HideMinimap">
        <property name="text">
         <string>Скрыть миникарту</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pb_ShowMinimap">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Показать миникарту</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="pbStep">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Сделать шаг</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_speedText">
        <property name="text">
         <string>Задать скорость</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGraphicsView" name="gv_Minimap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="FieldInfoBox" name="gbMapInformation"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mbMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Программа</string>
    </property>
    <addaction name="action_CreateNewField"/>
    <addaction name="action_LoadField"/>
    <addaction name="action_FigsLibrary"/>
    <addaction name="separator"/>
    <addaction name="action_CloseField"/>
    <addaction name="action_SaveFieldAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Run">
    <property name="title">
     <string>&amp;Симуляция</string>
    </property>
    <addaction name="action_OneStep"/>
    <addaction name="action_RunStop"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Rules"/>
   </widget>
   <widget class="QMenu" name="menu_Field">
    <property name="title">
     <string>П&amp;оле</string>
    </property>
    <addaction name="action_ClearField"/>
    <addaction name="action_AutofillField"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Field"/>
   <addaction name="menu_Run"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="MainFormStatusBar" name="sbMain"/>
  <action name="action_CreateNewField">
   <property name="icon">
    <iconset resource="../../res/mainresources.qrc">
     <normaloff>:/img/icons/new</normaloff>:/img/icons/new</iconset>
   </property>
   <property name="text">
    <string>Создать &amp;новую карту</string>
   </property>
   <property name="toolTip">
    <string>Заново создает новую карту</string>
   </property>
  </action>
  <action name="action_CloseField">
   <property name="icon">
    <iconset resource="../../res/mainresources.qrc">
     <normaloff>:/img/icons/close</normaloff>:/img/icons/close</iconset>
   </property>
   <property name="text">
    <string>Закрыть текущую карту</string>
   </property>
  </action>
  <action name="action_SaveFieldAs">
   <property name="icon">
    <iconset resource="../../res/mainresources.qrc">
     <normaloff>:/img/icons/save</normaloff>:/img/icons/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Сохранить как</string>
   </property>
  </action>
  <action name="action_LoadField">
   <property name="text">
    <string>&amp;Загрузить карту</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Выход</string>
   </property>
  </action>
  <action name="action_OneStep">
   <property name="text">
    <string>Сделать &amp;Шаг</string>
   </property>
  </action>
  <action name="action_RunStop">
   <property name="text">
    <string>Запустить/Остановить</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;О Программе</string>
   </property>
  </action>
  <action name="action_Rules">
   <property name="text">
    <string>&amp;Правила</string>
   </property>
  </action>
  <action name="action_ClearField">
   <property name="text">
    <string>&amp;Очистить от всего живого</string>
   </property>
  </action>
  <action name="action_AutofillField">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Автозаполнение</string>
   </property>
  </action>
  <action name="action_FigsLibrary">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Библиотека фигур</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainFormStatusBar</class>
   <extends>QStatusBar</extends>
   <header location="global">mainformstatusbar.h</header>
  </customwidget>
  <customwidget>
   <class>FieldInfoBox</class>
   <extends>QGroupBox</extends>
   <header location="global">fieldinfobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/mainresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStep</sender>
   <signal>clicked()</signal>
   <receiver>action_OneStep</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStop</sender>
   <signal>clicked()</signal>
   <receiver>action_RunStop</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRun</sender>
   <signal>clicked()</signal>
   <receiver>action_RunStop</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_HideMinimap</sender>
   <signal>clicked()</signal>
   <receiver>gv_Minimap</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_HideMinimap</sender>
   <signal>clicked()</signal>
   <receiver>pb_ShowMinimap</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_HideMinimap</sender>
   <signal>clicked()</signal>
   <receiver>pb_HideMinimap</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ShowMinimap</sender>
   <signal>clicked()</signal>
   <receiver>pb_ShowMinimap</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ShowMinimap</sender>
   <signal>clicked()</signal>
   <receiver>gv_Minimap</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ShowMinimap</sender>
   <signal>clicked()</signal>
   <receiver>pb_HideMinimap</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
